core/block_dragger.ts

```diff
@@ -445,6 +455,32 @@ export class BlockDragger implements IBlockDragger {
     eventUtils.setGroup(false);
   }
 
+  /**
+   * Moves the dragged block back to its original position before the start of
+   * the drag. Reconnects any parent and child blocks.\
+   */
+  private moveToOriginalPosition() {
+    this.startChildConn?.connect(this.draggingBlock_.nextConnection);
+    if (this.startParentConn) {
+      switch (this.startParentConn?.type) {
```

Comment:
You've already checked that `startParentConn` is defined, so you can take out the `?` here and below.

Reply:
Thanks for the catch!